package.json

```diff
@@ -384,4 +384,4 @@
     "node": "8.11.4",
     "yarn": "^1.10.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
__question:__ can you please remove unrelated changes like this one from this PR?